pkg/kubectl-explain/components/SlideInPanel.vue

```diff
@@ -0,0 +1,434 @@
+<script>
+import ExplainPanel from './ExplainPanel';
+import { KEY } from '@shell/utils/platform';
+import { expandOpenAPIDefinition, getOpenAPISchemaName, makeOpenAPIBreadcrumb } from '../open-api-utils.ts';
+
+const HEADER_HEIGHT = 55;
+
+export default {
+  components: { ExplainPanel },
+
+  props: {
+    schema: {
+      type:    Object,
+      default: () => {}
+    },
+    $t: {
+      type:     Function,
+      required: true,
+    }
+  },
+
+  data() {
+    return {
+      isOpen:         false,
+      definition:     undefined,
+      busy:           true,
+      error:          false,
+      expandAll:      false,
+      isResizing:     false,
+      resizeLeft:     '',
+      resizePosition: 'absolute',
+      width:          '33%',
+      right:          '-33%',
+      breadcrumbs:    undefined,
+      definitions:    {},
+      noResource:     false,
+      notFound:       false,
+    };
+  },
+
+  computed: {
+    top() {
+      const banner = document.getElementById('banner-header');
+      let height = HEADER_HEIGHT;
+
+      if (banner) {
+        height += banner.clientHeight;
```

Comment:
There's some slight weirdness with the footer. For this version, and given to slide in changes, don't think it's blocking

![image](https://github.com/rancher/dashboard/assets/18697775/98cfae19-d179-4f71-99e1-cf4e7b635e1f)